### DISCUSSION TITLE
Can't add repo

### DISCUSSION THREAD

**Discussion by danhogan** [SecKatie/ha-wyzeapi#652] [Q&A] [answered]
I'm getting the message "https://github.com/SecKatie/ha-wyzeapi is not a valid add-on repository"

**Comment by SobreroD** [comment]
i'm getting the same thing... hard to use if you can't add it

**Comment by okayjeffrey** [comment]
Same here

**Comment by brg468** [comment, marked as answer]
This is not a HA Addon. It needs to be installed through HACS or manually.
